proctor-tomcat-deps-provided/pom.xml

```diff
@@ -32,14 +32,6 @@
                     <groupId>org.apache.tomcat</groupId>
                     <artifactId>tomcat-jasper-el</artifactId>
                 </exclusion>
-                <exclusion>
```

Comment:
The `proctor-tomcat-deps` does not bring in `tomcat-jsp-api` and neither `tomcat-servlet-api`.